docs/releases/v0.23.0.md

```diff
@@ -0,0 +1,30 @@
+# Changelog
+
+## [v0.23.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.23.0) (2022-11-01)
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.22.0...v0.23.0)
+
+**Upstream release notes:**
+
+- [v0.62.1](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.62.1)
+- [v0.62.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.62.0)
+
+**Implemented enhancements:**
+
+- Requesting SumoLogic Exporter [\#1123](https://github.com/aws-observability/aws-otel-collector/issues/1123)
+
```

Comment:
```suggestion
```

This doesn't belong here.  It was closed as stale and not implemented.  We should fix this on the `release/v0.23.x` branch first.